doc/python-grader/index.md

```diff
@@ -0,0 +1,42 @@
+# Python Autograder
```

Comment:
Awesome to see docs for this. Could we also document the security model being used? That is, what runs as root, what runs as the `ag` user, etc?